package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
```

Comment:
The jump from `0.0.5` to `1.0.0` might be OK but might break things. Can we fix this by bumping `jest` instead?

Chain: `jest-cli` -> `jest-environment-jsdom` -> `jsdom` -> `request` -> `stringstream`